packages/design-core/docs/产品说明书.md

```
@@ -0,0 +1,196 @@
+
+### 2.前后对比
+
+原本支持固定状态切换的插件（仅物料和大纲树）：
+
+![image-20241008003823676](C:\Users\STATICHIT\AppData\Roaming\Typora\typora-user-images\image-20241008003823676.png)
+
+![image-20241008003835301](C:\Users\STATICHIT\AppData\Roaming\Typora\typora-user-images\image-20241008003835301.png)
+
+其它：
+
+![image-20241008003928410](C:\Users\STATICHIT\AppData\Roaming\Typora\typora-user-images\image-20241008003928410.png)	
```

Markdownlint

Column: 117
Hard tabs

(MD010, no-hard-tabs)